src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -234,7 +236,8 @@ type AdaptiveFSharpLspServer
   let sendDiagnostics (uri: DocumentUri) (diags: Diagnostic[]) =
     logger.info (Log.setMessageI $"SendDiag for {uri:file}: {diags.Length:diags} entries")
 
-    { Uri = uri; Diagnostics = diags } |> lspClient.TextDocumentPublishDiagnostics
+    // TODO: providing version would be very useful
+    { Uri = uri; Diagnostics = diags; Version = None } |> lspClient.TextDocumentPublishDiagnostics
```

Comment:
we don't flow versions through diagnostic reporting - we should investigate this

Reply:
I do remember doing this for work related to pull diagnostics but i can't find it. Probably better to start fresh anyway. Wanna fix this now or create an issue for later?

Reply:
This feels _big_ to do, so I'd rather log an item